### PR TITLE
Updated poco::dynamic::var to std::any in core.

### PR DESCRIPTION
**Pull request by parth-gr**
Check and edit these files ones for some common issues of DynamicObject:-
proxymodule.cpp
outputstream.cpp
Writesystemconfig.h
Writevariablegrid.cpp
Writesystemconfig.cpp
localdomaincontrollerbase.cpp
writevariablegrid.cpp
lookuprandomtransform.cpp
lookuptransform.cpp
sumpoolstransform.cpp
compositetransform.cpp

### CODE REVIEW

**Comment by parth-gr** [comment]
Signed-off-by: parth-gr 50455445+parth-gr@users.noreply.github.com

File: Source/moja.flint/include/moja/flint/proxymodule.h [hunk 1]
```diff
@@ -2,6 +2,7 @@
 #define MOJA_FLINT_PROXYMODULE_H_
 
 #include "moja/flint/moduleproxybase.h"
+#include <Poco/JSON/Parser.h>
```
Comment (malfrancis) [inline Source/moja.flint/include/moja/flint/proxymodule.h:5]: Should not be needed

File: Source/moja.flint/include/moja/flint/proxymodule.h [hunk 2]
```diff
@@ -10,10 +11,10 @@ class IVariable;
 // --------------------------------------------------------------------------------------------
 struct moduleProxyData {
    explicit moduleProxyData(const std::string& libraryName, std::string& moduleName, int order,
-                            std::shared_ptr<IModule> module, const DynamicObject& config);
+                            std::shared_ptr<IModule> module, const Poco::Dynamic::Var& config);
```
Comment (malfrancis) [inline Source/moja.flint/include/moja/flint/proxymodule.h:14]: Should be DynamicObject

File: Source/moja.flint/include/moja/flint/proxymodule.h [hunk 3]
```diff
@@ -10,10 +11,10 @@ class IVariable;
 // --------------------------------------------------------------------------------------------
 struct moduleProxyData {
    explicit moduleProxyData(const std::string& libraryName, std::string& moduleName, int order,
-                            std::shared_ptr<IModule> module, const DynamicObject& config);
+                            std::shared_ptr<IModule> module, const Poco::Dynamic::Var& config);
 
    explicit moduleProxyData(const std::string& libraryName, std::string& moduleName, int order,
-                            std::shared_ptr<IModule> module, const DynamicObject& config, std::string variableName,
+                            std::shared_ptr<IModule> module, const Poco::Dynamic::Var& config, std::string variableName,
```
Comment (malfrancis) [inline Source/moja.flint/include/moja/flint/proxymodule.h:17]: should be DynamicObject

File: Source/moja.flint/src/outputerstream.cpp [hunk 4]
```diff
@@ -17,6 +17,11 @@
 #include <iomanip>  // std::setprecision
 #include <iostream>
 
+
+#include <Poco/JSON/Parser.h>
```
Comment (malfrancis) [inline Source/moja.flint/src/outputerstream.cpp:21]: should not be needed

File: Source/moja.flint/src/outputerstream.cpp [hunk 5]
```diff
@@ -17,6 +17,11 @@
 #include <iomanip>  // std::setprecision
 #include <iostream>
 
+
+#include <Poco/JSON/Parser.h>
+#include "moja/dynamic.h"
+#include "Poco/Dynamic/Struct.h"
```
Comment (malfrancis) [inline Source/moja.flint/src/outputerstream.cpp:23]: should not be needed

File: Source/moja.flint/src/outputerstream.cpp [hunk 6]
```diff
@@ -39,7 +44,7 @@ std::string escape_json2(const std::string& s) {
 
 // --------------------------------------------------------------------------------------------
 
-void outputDynamicToStream(std::ostream& fout, const DynamicVar& object) {
+void outputDynamicToStream(std::ostream& fout, const Poco::Dynamic::Var& object) {
```
Comment (malfrancis) [inline Source/moja.flint/src/outputerstream.cpp:47]: should be DynamicVar

File: Source/moja.flint/src/proxymodule.cpp [hunk 7]
```diff
@@ -6,13 +6,15 @@
 #include <moja/notificationcenter.h>
 #include <moja/signals.h>
 
+#include <Poco/JSON/Parser.h>
```
Comment (malfrancis) [inline Source/moja.flint/src/proxymodule.cpp:9]: should not be needed

File: Source/moja.flint/src/proxymodule.cpp [hunk 8]
```diff
@@ -6,13 +6,15 @@
...
+
 namespace moja {
 namespace flint {
 
 // --------------------------------------------------------------------------------------------
 
 moduleProxyData::moduleProxyData(const std::string& libraryName, std::string& moduleName, int order,
-                                 std::shared_ptr<IModule> module, const DynamicObject& config) {
+                                 std::shared_ptr<IModule> module, const Poco::Dynamic::Var& config) {
```
Comment (malfrancis) [inline Source/moja.flint/src/proxymodule.cpp:17]: should be DynamicObject

File: Source/moja.flint/src/proxymodule.cpp [hunk 9]
```diff
@@ -40,7 +43,7 @@ moduleProxyData::moduleProxyData(const std::string& libraryName, std::string& mo
    _module->subscribe(*_notificationCenterPtr.get());
 }
 
-void ProxyModule::configure(const DynamicObject& config) {
+void ProxyModule::configure(const Poco::Dynamic::Var& config) {
```
Comment (malfrancis) [inline Source/moja.flint/src/proxymodule.cpp:46]: should be DynamicObject

File: Source/moja.flint/src/variableandpoolstringbuilder.cpp [hunk 10]
```diff
@@ -10,6 +10,8 @@
 
 #include <boost/format.hpp>
 
+#include <Poco/JSON/Parser.h>
```
Comment (malfrancis) [inline Source/moja.flint/src/variableandpoolstringbuilder.cpp:13]: should not be needed

File: Source/moja.flint/src/variableandpoolstringbuilder.cpp [hunk 11]
```diff
@@ -145,7 +147,7 @@ std::string escape_json2_XX(const std::string& s) {
 
 // --------------------------------------------------------------------------------------------
 
-void outputDynamicToStream_XX(std::ostream& fout, const DynamicVar& object) {
+void outputDynamicToStream_XX(std::ostream& fout, const Poco::Dynamic::Var& object) {
```
Comment (malfrancis) [inline Source/moja.flint/src/variableandpoolstringbuilder.cpp:150]: shoud be DynamicVar

**Review by malfrancis — CHANGES_REQUESTED** [review]
This pull request has pulled in several commits from previous check-ins these should be removed with just the latest changes. I'm not sure how this happened. 
Poco::Dynamic should be replaced and there should be no need for Poco::JSON

**Comment by parth-gr** [comment]
> This pull request has pulled in several commits from previous check-ins these should be removed with just the latest changes. I'm not sure how this happened.
> Poco::Dynamic should be replaced and there should be no need for Poco::JSON

Okay, sir, I will update the pull request and I will update the Poco::Dynamic.

File: Source/moja.flint/src/variableandpoolstringbuilder.cpp (hunk 11 above)
Comment (parth-gr) [reply Source/moja.flint/src/variableandpoolstringbuilder.cpp]: Done

**Comment by parth-gr** [comment]
Sir please the updation I have done.
I am not getting the pervious commits how they are coming, sir, you may please look on this repo: https://github.com/parth-gr/FLINT/commits/feature/profile I have given the access to you.

I tried git-rebase, still figuring if I can remove those commits.

**Comment by parth-gr** [comment]
@malfrancis Sir in this PR, Like I was able to remove the last commits, but it's like giving an overhead of again and again commits, so should I provide a new PR?

**Pull request closed without merging**